include/seadsa/CallGraphUtils.hh

```diff
@@ -70,16 +70,16 @@ SortedCallSites(llvm::CallGraphNode *cgn,
             : *callee_func;
 
     if (callee.isDeclaration() || callee.empty()) continue;
-    res.push_back(callRecord.first);
+    res.push_back(*callRecord.first);
```

Comment:
this deref looks suspicious. What is the type of `callRecord.first`?

Reply:
It's a `Optional<WeakTrackingVH>`(https://llvm.org/doxygen/classllvm_1_1CallGraphNode.html#afcbc00d65e70c40cfa453cd8164b38f7).